src/storage/src/hummock/vacuum.rs

```diff
@@ -35,22 +35,16 @@ impl Vacuum {
         sstable_store: SstableStoreRef,
     ) -> HummockResult<VacuumTask> {
         tracing::info!("Try to vacuum SSTs {:?}", vacuum_task.sstable_object_ids);
-        let object_ids = vacuum_task.sstable_object_ids;
         sstable_store
-            .delete_list(&object_ids)
-            .await
-            .map_err(|e| HummockError::meta_error(format!("Failed to vacuum task: {:#?}", e)))?;
-        let vacuum_task = VacuumTask {
-            sstable_object_ids: object_ids,
-        };
+            .delete_list(&vacuum_task.sstable_object_ids)
```

Comment:
How about passing vacuum_task.sstable_object_ids directly? This avoids having to return the vacuum_task again.